README.md

```diff
@@ -1,24 +1,124 @@
 
-* Database initialization
+Put React in ruby on rails app
+## 🛠 Built With <a name="built-with"></a>
 
-* How to run the test suite
+### Tech Stack <a name="tech-stack"></a>
 
-* Services (job queues, cache servers, search engines, etc.)
+<details>
+<summary>Ruby</summary>
+  <ul>
+    <li><a href="https://www.ruby-lang.org/es/">Ruby</a></li>
+    <li><a href="https://github.com/github/rubocop-github/blob/main/STYLEGUIDE.md">Ruby styles guide</a></li>
+    <li><a href="https://www.ruby-lang.org/es/">Rails</a></li>
+  </ul>
+</details>
 
-* Deployment instructions
+### Key Features <a name="key-features"></a>
 
-* ...
+- **Use Gitflow**
+- **Implement classes and objects**
+- **Use encapsulation and inheritance**
+- **Ruby styles reviewed with Rubocop Linters**
+- **Use composition**
+- **Ruby on rails**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+<!-- LIVE DEMO -->
+## 🚀 Link for front end app repository <a name="live-demo"></a>
+>
+https://github.com/francksefu/hello_rails/pull/1
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+This repository includes files with ruby lenguage.
+
+### Prerequisites
+
+In order to run this project you need to have Ruby installed, check [this](https://www.ruby-lang.org/en/) documentation in order to know how to install it.
+-Install Rails
+
+### Setup
+
+Clone this repository to your desired folder:
+
+  cd my-folder https://github.com/francksefu/hello_rails_react.git
+  git clone 
+
+### Usage
+
+In this project you are allow to add post , add comment on post and like post
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Author <a name="authors"></a>
+
+👤 **Franck Sefu**
+
+- GitHub: [francksefu](https://github.com/francksefu)
+- Twitter: [francksefu](https://twitter.com/franck_sefu)
+- LinkedIn: [francksefu](https://www.linkedin.com/in/franck-sefu/)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **Create table by using rails commands**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/francksefu/hello_rails_react/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project you could clone the repo and work with it or do the changes in a new branch. Also you could let a message with your thoughts.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Microverse for providing the necessary resources for this project.
+I wanna give thanks to [Gregoire V](https://www.behance.net/gregoirevella), the [author of the original design]
+
```

Comment:
- [ ] I think it would be great if you removed this since this is from the capstone project and you don't need it to give the acknowledgment here to Grgoire. 🔥🙌